bindings/python/Makefile.am

```diff
@@ -33,7 +33,7 @@ EXTRA_DIST = \
 if ENABLE_PYTHON_BINDING
 
 all-local:
-	$(PYTHON) $(srcdir)/setup.py build
+	CXXFLAGS="$(CXXFLAGS)" $(PYTHON) $(srcdir)/setup.py build
```

Comment:
But AX_CXX_COMPILE_STDCXX_11 would not include the flags which are configured with configure.ac. For example `-ftemplate-depth=128 -Wno-format-zero-length"` would be missing in python because we leave the make scope by calling $(PYTHON)
I would guess we may need even more then CXXFLAGS="$(CXXFLAGS)" instead of less, isn't it?

And if I use AX_CXX_COMPILE_STDCXX_11 I would need to write
`CXX="$(CXX)" $(PYTHON) $(srcdir)/setup.py build`
because it add the flag to CXX https://github.com/sandstorm-io/capnproto/blob/master/c%2B%2B/m4/ax_cxx_compile_stdcxx_11.m4 and python does not see it because it is not in make scope